lib/spirv/lib/src/transpiler.dart

```diff
@@ -77,6 +77,12 @@ class _Transpiler {
   /// ID mapped to ID. Used by [OpLoad].
   final Map<int, int> alias = <int, int>{};
 
+  /// Set of IDs for constant true values.
+  final Set<int> constantTrues = <int>{};
```

Comment:
Should only need one for each of these - `int constantTrue = 0;` I think the spec isn't super clear on this.. but in the interest of speed/memory efficiency I think using one int is better, unless we actually see multiple-definitions of opConstantTrue/False in the wild later on.

Reply:
Sounds good